microprofile-jwt-auth-tool/src/main/java/org/jboss/eap/qe/microprofile/jwt/auth/tool/JsonWebToken.java

```diff
@@ -0,0 +1,113 @@
+package org.jboss.eap.qe.microprofile.jwt.auth.tool;
+
+import java.io.UnsupportedEncodingException;
+import java.security.GeneralSecurityException;
+import java.security.PrivateKey;
+import java.security.Signature;
+import java.util.Base64;
+
+/**
+ * A class representing JsonWebToken
```

Comment:
This is kind of needless, isn't it? Class name is pretty self-descriptive.

Reply:
Well, it is kind of wrapper, though if you require removal of this comment, I will comply.